### DISCUSSION TITLE
osu!lazer not launching (sometimes) from .NET application error

### DISCUSSION THREAD

**Discussion by SS374** [ppy/osu#19700] [Q&A] [answered]
running osu is a 50/50 chance of launching and not launching. sometimes it does, and when it doesn't, it won't launch until i restart my computer multiple times. sometimes it does and then doesn't in the same desktop use session. don't wanna restart my laptop every time i want to play funny circle clicking game

in all seriousness though, here are some information
Task Manager not showing that osu!lazer is running in the foreground or background. 
When it doesn't launch, I'm not saying it launch and crashes. Nothing happens.
Event Viewer reports a Error level .NET application failure, filepath C:\Users\[name]\AppData\Local\osulazer\app-2022.810.2\osu!.exe, messsage "Failed to create CoreCLR, HRESULT: 0x80004005". 
see more below:

osu logs:
[runtime.log](https://github.com/ppy/osu/files/9305663/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9305664/updater.log)
[database.log](https://github.com/ppy/osu/files/9305665/database.log)
[performance.log](https://github.com/ppy/osu/files/9305666/performance.log)

more info:
i7 11th gen 3080m laptop
Windows 10 21H2
crashing started months ago, even with new updates and such. have even reinstalled Windows 10 and osu various times but it never solves the not launching issue

i'm on the latest version (can't check right now because it can't launch) currently, just installed a update today (when it was working).

**Comment by peppy** [comment, marked as answer]
https://github.com/dotnet/core/issues/6733

Looks like updating to windows 11 will fix.

If that's not an option, you will need to wait for a backport to windows 10. There's no guarantee that will happen.
